Adding error in this troubleshooting (#1724)
vtexdocs/dev-portal-content: docs/localization/i-cant-install-the-headless-cms-plugins-in-my-account.md

---
title: "I can’t install the Headless CMS plugins in my account"

> excerpt: "Windows users may encounter the 'EPERM: operation not permitted' error during the installation of the Headless CMS plugin."

Reviewer: Como agora são dois problemas relacionados a instalação, talvez deixar um pouco mais genérico aqui. Algo como "Users may encounter errors related to symbolic links and node version incompatibility."

slug: "error-installing-headless-cms-plugins-on-windows"
updatedAt: "2025-02-12T14:56:25.977Z"
tags:
    - headless-cms
    - faststore
---

> If you are a Windows user, you may encounter the following errors after [installing the Headless CMS plugin](https://developers.vtex.com/docs/guides/faststore/headless-cms-1-configuring-the-vtex-account#step-1-setting-up-the-command-line-environment).

Reviewer: ```suggestion
FastStore users may encounter the following errors after [installing the Headless CMS plugin](https://developers.vtex.com/docs/guides/faststore/headless-cms-1-configuring-the-vtex-account#step-1-setting-up-the-command-line-environment).
```

Reviewer: Como o último erro pode ser tanto para usuário do Windows como IOS, uma alternativa é deixar mais genérica essa parte.

| `Error: EPERM: operation not permitted, symlink`                              | Your device isn’t enabled for development.                                     | [Enable Developer Mode in your machine](https://developers.vtex.com/docs/troubleshooting/error-installing-headless-cms-plugins-on-windows#enable-developer-mode-in-your-machine) |
| `error @oclif/core@1.26.2: The engine "node" is incompatible with this module. Expected version ">=14.0.0". Got "12.12.0"` | The VTEX IO CLI’s node version isn’t updated. Expected version was `>=14.0.0`, but the current version is `12.12.0`. | [Update VTEX IO CLI](https://developers.vtex.com/docs/troubleshooting/error-installing-headless-cms-plugins-on-windows#update-vtex-io-cli) |

## Solutions

To solve these issues, see the following instructions:

- [Enable Developer Mode on your machine](#enable-developer-mode-on-your-machine)
- [Update VTEX IO CLI](#update-vtex-io-cli)

### Enable Developer Mode on your machine

> If you run the `vtex plugins install cms` and receive the error below, you must enable the Developer Mode on your machine.

Reviewer: ```suggestion
Windows users, while running the `vtex plugins install cms`, can receive the error below:
```

```sh
vtex plugins install cms
warning "@vtex/cli-plugin-cms > @vtex/api > apollo-server-core > apollo-graphql@0.9.5" has incorrect peer dependency "graphql@^14.2.1 || ^15.0.0".
Installing plugin @vtex/cli-plugin-cms... failed
    Error: EPERM: operation not permitted, symlink
    'C:\Users\LukeSkywalker\AppData\Roaming\npm\node_modules\vtex' ->
    'C:\Users\LukeSkywalker\AppData\Local\vtex\node_modules\vtex'
    Code: EPERM
```

> FastStore projects rely on creating symbolic links (symlinks), and the [Developer Mode](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development#accessing-settings-for-developers) grants the necessary permissions and privileges to use them, which reduces the probability of encountering errors during development.

Reviewer: ```suggestion
FastStore projects rely on creating symbolic links (symlinks), and to solve this error, you must enable the [Developer Mode](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development#accessing-settings-for-developers) on your machine. The Developer Mode grants the necessary permissions and privileges to use the symlinks, which reduces the probability of encountering errors during development.
```
